tasks/windows.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Manage the state of Windows services (without a puppet agent)",
```

Comment:
Is the "Without a puppet agent" needed?  Isn't bolt implicitly sans-puppet

Reply:
Nope... same tasks can be run through PE tasks support via pxp-agent OR via Bolt. The other existing `init.sh` task in this module uses the Puppet agent.

Also, I did crib that bit from the TP PR at #23